tests/test_dataflow/core/test_linearize.py

```diff
@@ -97,31 +96,22 @@ def test_unnest_line():
 def test_sexp_formatter_and_deformatter():
     """Round-trip tests for s-expression formatter and deformatter."""
     data = [
-        ("", ""),
-        ('# ( String "singleToken" )', '# ( String " singleToken " )'),
-        ('# ( String "multiple tokens" )', '# (  String " multiple tokens " )'),
-        # these two are not valid value s-expressions, so they are not formatted
-        ('# ( String "open quote only )', '# (  String "open quote only )'),
-        ('# ( String closing quote only" )', '# ( String closing quote only" )',),
-        # multiple value s-expressions
```

Comment:
Not sure why were were testing these things.

Reply:
Are they broken with the new changes? If not, maybe we can just keep them.
I think these were intended to test corner cases. They may have appeared in some version of the data before, although not the current versions.

Reply:
Yeah, because the formatting operates on parsed s-expressions and these are not valid s-expressions. I don't think we had any invalid s-expressions in the old data?

Reply:
Got it. Then removing them is fine.